Voycar.Api.Web/Features/Plans/Endpoints/Put/Personal/Validator.cs

```diff
@@ -0,0 +1,14 @@
+namespace Voycar.Api.Web.Features.Plans.Endpoints.Put.Personal;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
+    {
+        this.RuleFor(request => request.PlanId)
+            .NotEmpty()
+            .WithMessage("Plan id is required")
+            .WithName("generalErrors");
```

Comment:
```suggestion
        this.RuleFor(request => request.PlanId)
            .NotEmpty()
            .WithMessage("Plan id is required")
            .WithName("generalErrors");
```
Ich glaube der Validator kann komplett weg. Diese eine Regel, die er hat, wird schon dadurch abgedeckt, dass in `HandleAsync` gecheckt wird, ob ein `Plan` mit der gegebenen ID in der Datenbank ist.